src/dev/i18n/README.md

```diff
@@ -0,0 +1,208 @@
+# I18n Build Tools
```

Comment:
not in scope of this PR, but I would move each tool for separate folder, like: `exracting`, `verification`, `utils` (maybe for common helpers) etc.